tencentcloud/resource_tc_mysql_instance.go

```diff
@@ -1159,7 +1205,7 @@ func mysqlUpdateInstancePayByMonth(ctx context.Context, d *schema.ResourceData,
 		d.SetPartial("auto_renew_flag")
 	}
 
-	if d.HasChange("period") {
+	if d.HasChange("period") || d.HasChange("prepaid_period") {
```

Comment:
fmt.Errorf("After the initialization of the field[%s] to set does not make sense", "period or prepaid_period")